src/ontology/metadata/omim-metrics.json

```diff
@@ -4,10 +4,10 @@
     "abox_axiom_count_incl": 0,
     "annotation_property_count": 18,
     "annotation_property_count_incl": 18,
-    "axiom_count": 345057,
-    "axiom_count_incl": 345057,
-    "class_count": 19439,
-    "class_count_incl": 19439,
+    "axiom_count": 345359,
+    "axiom_count_incl": 345359,
+    "class_count": 19458,
+    "class_count_incl": 19458,
```

Comment:
in these "metrics" files, what is the source being evaluated/reviewed for these metrics?

Reply:
The component not the exact source. You can see the corresponding Makefile goal here:
https://github.com/monarch-initiative/mondo-ingest/blob/2fdc6dfa88520094c5765b8fe00fe25a41d387ed/src/ontology/mondo-ingest.Makefile#L318